docs/business-apps/power-automate/sharepoint-connector-actions-triggers.md

```diff
@@ -27,6 +27,8 @@ Triggers the flow when you delete an item in a list. To get the properties of a
 
 Triggers the flow when you select an item in a list.
 
+**Note:** Only flows within the default environment can be executed manually from a SharePoint list or library.
```

Comment:
Please resubmit using the MS Docs approved "note" call alert callout as used in other docs such as this one: 
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-developer-tenant